main.go

```diff
@@ -276,19 +403,48 @@ func createMilestoneData(advance int, timeInterval string) map[string]milestone
 	return milestones
 }
 
-func createMilestones(baseURL string, token string, projectID string, milestones map[string]milestone) error {
+func createMilestones(baseURL string, token string, projectID string, milestones map[string]milestone, api string) error {
 	client := &http.Client{}
-	strURL := []string{baseURL, "/projects/", projectID, "/milestones"}
+	var strURL []string
+	switch {
+	case api == "gitlab":
+		strURL = []string{baseURL, "/projects/", projectID, "/milestones"}
+	case api == "github":
+		strURL = []string{baseURL, "/milestones"}
```

Comment:
Looks a bit weird to add `"/projects/"` here for gitlab, but not for github where it's done before the call to these functions.

Reply:
Yeah, it's a result of the way the URL is formed differently between the two APIs. GitHub has milestones for issues and not projects, so there isn't a need to get the project ID like we do for GitLab.